osu.Game.Tournament/Screens/Gameplay/GameplayScreen.cs

```diff
@@ -36,18 +41,41 @@ public class GameplayScreen : BeatmapInfoScreen, IProvideVideo
 
         private Drawable chroma;
 
+        private TourneyVideo fallbackGameplayVideo;
+        private readonly List<(string, TourneyVideo)> modSpecificGameplayVideos = new List<(string, TourneyVideo)>();
+
         [BackgroundDependencyLoader]
-        private void load(LadderInfo ladder, MatchIPCInfo ipc, Storage storage)
+        private void load(LadderInfo ladder, MatchIPCInfo ipc, TournamentStorage storage)
         {
             this.ipc = ipc;
 
+            var extraGameplayVideos = new List<TourneyVideo>();
+
+            // look for all video files that start with "gameplay-"
+            foreach (var name in storage.GetFiles("videos", "gameplay-*").Select(x1 => Path.GetFileNameWithoutExtension(x1[("videos".Length + 1)..])))
+            {
+                var vid = new TourneyVideo(name)
+                {
+                    Loop = true,
+                    RelativeSizeAxes = Axes.Both,
+                    Alpha = 0
+                };
+                extraGameplayVideos.Add(vid);
+                modSpecificGameplayVideos.Add((name["gameplay-".Length..], vid));
```

Comment:
would be probably better to write
```suggestion
                modSpecificGameplayVideos.Add((name.Replace("gameplay-", ""), vid));
```

Reply:
I guess idk, to me in stipping away the first bit of the string, so to me what I did feels more correct, but I dont really mind either way tbh. Dunno if anyone else has more thoughts on this?

Reply:
The syntax you used is very hard to understand, so you'd need to use `Replace` or `Substring`.

Reply:
fwiw my inspiration to use this syntax came from https://github.com/ppy/osu-framework/blob/1abee4427957340880e2f69044f535dcac61978c/osu.Framework/IO/Stores/NamespacedResourceStore.cs#L30, personally I think its quite intuitive.
Anyway I'll change it, I dont have a strong opinion either way